cmd/collect.go

```diff
@@ -56,12 +55,21 @@ func collectCommand(c *cli.Context) error {
 		return err
 	}
 
+	err = collect(ctx, api, runner, id, output)
```

Comment:
Extracting this previously didn't make sense as we were literally getting a file as the response from the API.

However now the contract has changed and we actually get a JSON response from the `collect` API endpoint, so makes sense to extract together with the parsing part.